### ISSUE TITLE
Version 1.2.4 libs bad idea

### ISSUE THREAD

**Issue by manguz** [Magisk-Modules-Repo/google-framework-magisk#7]
Overriding system libs in a module for all devices is a very bad idea, and will break a lot of things and devices.
Just so you know

**Comment by sirleon23** [comment]
Manguz do you have the Second last update? I need to roll back to the previous version

**Comment by sirleon23** [comment]
This update breaks WiFi on op3t

**Comment by manguz** [comment]
And will break my face unlock on my huawei devices probably.
Libs are not the same in various systems and you should not replace it.

**Comment by sirleon23** [comment]
Hope some one has the previous version to come back again 🙄🙄🙄🙄

**Comment by manguz** [comment]
Which one you want?
Give me your mail

**Comment by sirleon23** [comment]
Sebarena780@gmail.c0m
Thank you man
Honestly I don't remember which version I had (1.2.1/2/3) probably the first one will be the best in my situation

**Comment by emaf94** [comment]
v1.2.4 makes bootloop in my g5 plus, the v1.2.3 worked perfect

**Comment by RAYdss** [comment]
Alguém pode me fornecer a versão 1.2.3

**Comment by soham2008xyz** [comment]
Version 1.2.4 caused a bootloop on my Moto G4 (athene) running stock nougat (7.0) rooted with magisk.

Last stable version AFAIK is 1.2.2. I have not tried out 1.2.4.

**Comment by BROKEN81** [comment]
Manzue can you give me the older version?

**Comment by Hyperbomb64** [comment]
Yeah I'm the same as others here. Just updated to 1.2.4 today and caused bootloop. I believe the last version I was on that worked was 1.2.2 so if anyone has that it would be appreciated. Also willing to give 1.2.3 a try.

**Comment by BROKEN81** [comment]
https://forum.xda-developers.com/moto-x-style/themes-apps/google-dialer-running-nougat-t3698417

Flash this instead
